chart/templates/persistentvolumeclaim.yml

```diff
@@ -18,7 +19,8 @@ spec:
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: valheim-server-base-data
+  name: {{ .Release.Name }}-server-base-data
```

Comment:
This is also a breaking change, even when the Helm release name was `valheim-server` before. In this case, the old PVC would have been named `valheim-server-base-data`  and the new one with the same release name `valheim-server-server-base-data`.